app/server.hs

```diff
@@ -11,6 +11,7 @@ options = Settings
   <*> option auto (long "port" <> value defaultPort <> help "Port number")
   <*> strOption (long "live" <> value "." <> metavar "DIR" <> help "Live prefix")
   <*> optional (strOption (long "archive" <> metavar "DIR" <> help "Archive prefix"))
```

Comment:
Can you change the help menu to reflect the fall-back behaviour? Something like

```
If the archive prefix does not exist, a live stream is used automatically.
```

Reply:
`s/the archive prefix/the archive path/`?

Reply:
Sure.